Add `HistoricalSummaries` to beacon chain network specs (#210)
ethereum/portal-network-specs: beacon-chain/light-client-network.md

> # Beacon Chain Network

Reviewer: If we rename the network, the name of the file should also change.

Reviewer: Perhaps just leave the name as Beacon Chain LC network for now.

Author: I think I'm going to keep for now the network name  `Beacon Chain Network`  because the data that we are going to store is not only light client data. I'll rename the file in a separate PR to keep the current changes clean.

```py
# Maximum number of `LightClientUpdate` instances in a single request
MAX_REQUEST_LIGHT_CLIENT_UPDATES = 2**7  # = 128

# Maximum number of `HistoricalSummary` records

> HISTORICAL_ROOTS_LIMIT = 2**24  # = 16,777,216

Reviewer: See comment below, but this doesn't need to be redefined and can be referenced from consensus spec.

content_key                          = selector + SSZ.serialize(light_client_optimistic_update_key)
```

> A `0` in the content key is equivalent to the request for the latest
LightClientOptimisticUpdate that the requested node has available.

#### HistoricalSummaries

> Latest HistoricalSummaries are stored in the network every 256 epochs (8192 slots).

Reviewer: ```suggestion
Latest `HistoricalSummaries` object is stored in the network every 256 epochs (8192 slots).
```

I think this is what you meant, just adjusting the text a bit to make it clear that a node only is required to store 1.
